finals-2024-01-15/code/5345-daniel-naumoski/src/app/footer/footer.component.ts

```diff
@@ -1,11 +1,11 @@
 import { Component, Input } from '@angular/core';
-import { Student } from '../models/student';
+import { Movie } from '../models/Movie';
```

Comment:
+4, not really sure why we need a movie input into the footer (it's not used anyway)